tests/projects/test_project_tags.py

```diff
@@ -59,71 +64,131 @@ def test_is_a4_project(project, external_project, bplan):
 
 
 @pytest.mark.django_db
-def test_get_num_entries(
+def test_render_module_insights(
     django_assert_num_queries,
     module_factory,
     phase_factory,
     idea_factory,
     proposal_factory,
     comment_factory,
+    live_question_factory,
+    topic_factory,
+    rating_factory,
+    chapter_factory,
+    poll_factory,
+    question_factory,
+    choice_factory,
+    vote_factory,
 ):
-    phase, module, project, idea = setup_phase(
-        phase_factory, idea_factory, idea_phases.FeedbackPhase
+    phase_ic, module_ic, project_ic, idea = setup_phase(
+        phase_factory,
+        idea_factory,
+        idea_phases.FeedbackPhase,
+        module__blueprint_type="IC",
     )
+
     phase1, module1, project1, proposal = setup_phase(
-        phase_factory, proposal_factory, budgeting_phases.CollectPhase
+        phase_factory,
+        proposal_factory,
+        budgeting_phases.CollectPhase,
+        module__blueprint_type="PB",
+    )
+
+    phase_ie, module_ie, project_ie, question = setup_phase(
+        phase_factory,
+        live_question_factory,
+        livequestion_phases.IssuePhase,
+        module__blueprint_type="IE",
+    )
+
+    phase_tp, module_tp, project_tp, topic = setup_phase(
+        phase_factory,
+        topic_factory,
+        topicprio_phases.PrioritizePhase,
+        module__blueprint_type="TP",
+    )
+
+    phase_tr, module_tr, project_tr, chapter = setup_phase(
+        phase_factory,
+        chapter_factory,
+        textreview_phases.CommentPhase,
+        module__blueprint_type="TR",
+    )
+
+    phase_po, module_po, project_po, poll = setup_phase(
+        phase_factory,
+        poll_factory,
+        voting_phases.VotingPhase,
+        module__blueprint_type="PO",
     )
-    empty_module = module_factory(project=project)
+
+    empty_module = module_factory()
 
     # no items or comments
     clear_query_cache()
-    with django_assert_num_queries(18):
-        count = get_num_entries(empty_module)
-        assert count == 0
+    with django_assert_num_queries(0):
+        data = render_module_insights(empty_module)
+        assert data["count"] == 0
+        assert data["label"] == ngettext("Idea", "Ideas", 0)
+        assert data["icon"] == "far fa-lightbulb"
```

Comment:
what's the empty module default? it's an idea? hence the icon and the label for ideas? if so can you add an inline comment when you create the empty_module.